spec/stringProcessing/countSyllableSpec.js

```diff
@@ -23,3 +23,92 @@ describe( "counting syllables", function() {
 		expect( countSyllableFunction( "strawberry" ) ).toBe( 3 );
 	})
 });
+
+describe("a syllable counter for Dutch textstrings", function(){
```

Comment:
I'm noticing a mixture of two different code styles. Please use a uniform code style.